### PR TITLE
Add support for release channels configuration for SQL Endpoints

### PR DESCRIPTION
**Pull request by alexott**

### CODE REVIEW

**Comment by alexott** [comment]
The only one thing could be potentially required - right now, when you change channel, it's applied momentarily, without restart. But I'm not sure that it's correct behavior - checking this with developers

**Review by nfx — COMMENTED** [review]
Please add a change log entry, otherwise lgtm:)

**Comment by alexott** [comment]
done

**Review by nfx — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## 0.4.8
 
 * Added support for `tf:suppress_diff` on primitive types ([#984](https://github.com/databrickslabs/terraform-provider-databricks/issues/984)).
+* Added support for release channels in SQL Endpoints configuration ([#1078])(https://github.com/databrickslabs/terraform-provider-databricks/pull/1078).
```
Comment (nfx) [inline CHANGELOG.md:6]: SQL endpoints -> `databricks_sql_endpoint`, for consistency

**Comment by nfx** [comment]
there's -0.63% coverage change, even though not directly related to your change. can you also please cover one of the misses to bring coverage back - https://app.codecov.io/gh/databrickslabs/terraform-provider-databricks/blob/master/sqlanalytics/resource_sql_endpoint.go

File: CHANGELOG.md (hunk 1 above)
Comment (alexott) [reply CHANGELOG.md:6]: done

**Comment by nfx** [comment]
@alexott what about coverage? :)

**Comment by alexott** [comment]
why it decreased at all? Will look later today when get out of meetings

**Comment by alexott** [comment]
ok, it's better now

**Pull request merged**